setup.py

```diff
@@ -123,7 +123,8 @@ def get_flash_attention_extensions(cuda_version: int, extra_compile_args):
     for arch in archs_list.replace(" ", ";").split(";"):
         assert len(arch) >= 3, f"Invalid sm version: {arch}"
 
-        num = 10 * int(arch[0]) + int(arch[2])
+        arch_arr = arch.split('.')
+        num = 10 * int(arch_arr[0]) + int(arch_arr[1])
```

Comment:
Oh actually, I'm not sure how this will work with the `+PTX` suffix (eg `7.0;8.0+PTX`). Can you address that as well?

Reply:
Good call.  Patch incoming.